integration_tests/cli/pki-noc-certs.sh

```diff
@@ -7,6 +7,9 @@ noc_root_cert_1_subject_key_id="44:EB:4C:62:6B:25:48:CD:A2:B3:1C:87:41:5A:08:E7:
 noc_root_cert_1_serial_number="47211865327720222621302679792296833381734533449"
 noc_root_cert_1_subject_as_text="CN=NOC-1,OU=Testing Division,O=Example Company,L=Tashkent,ST=Some State,C=UZ"
 
+noc_root_cert_1_copy_path="integration_tests/constants/noc_root_cert_1_copy"
```

Comment:
After discussions, it was decided to do it in the scope of another task since we have the same certificates types but with a different naming format, it will be necessary to convert the names into a common format

Reply:
> I think that using 'copy' as a postfix might not be the best choice, maybe it would be better to add a part of the serial number as a postfix instead?

After discussions, it was decided to do it in the scope of another task since we have the same certificates types but with a different naming format, it will be necessary to convert the names into a common format